File: esercizi_java/NetBeansProjects/Circolo_Sportivo/src/circolo_sportivo/Campo.java
package circolo_sportivo;

/**
 *
 * @author tobiasacchetto
 */
public class Campo {
    protected int cod;
    private String nome;
    protected float lunghezza, larghezza, costorario;
    public Campo(int cod,String nome,float larghezza, float lunghezza, float costorario){
        this.cod= cod;
        this.nome=nome;
        this.larghezza= larghezza;
        this.lunghezza = lunghezza;
        this.costorario = costorario;
    }
    
    public String toString(){
        return nome+"\t"+cod+"\t"+larghezza+"\t"+lunghezza+"\t";
    }
}

File: esercizi_python/compito_python_3/Campo.py
class Campo:
    def __init__(self, cod, nome, larghezza, lunghezza, costorario):
        self.cod = cod
        self.nome = nome
        self.larghezza = larghezza
        self.lunghezza = lunghezza
        self.costorario = costorario

    def toString(self):
        return self.nome+"\t"+str(self.cod)+"\t"+str(self.larghezza)+"\t"+str(self.lunghezza)+"\t"
